sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/config/constants/OpenApiConstants.java

```diff
@@ -27,10 +27,10 @@ interface SecuritySchemes {
   }
 
   interface Tags {
-
     String NAMESPACES = "Namespaces";
     String COLLECTIONS = "Collections";
     String DOCUMENTS = "Documents";
+    String JSON_SCHEMAS = "Json Schemas";
```

Comment:
can you add a description for this tag in the `StargateDocApi` class.. And if you  don't mind for the `NAMESPACES` as well, as I forgot to do it.. Once we have a description it's looking nicer in the Swagger:

![image](https://user-images.githubusercontent.com/10600041/172118401-80eed393-2071-4c1f-81b6-14906a8c0d5e.png)